tempesta_fw/http_msg.c

```diff
@@ -46,7 +46,7 @@ tfw_http_msg_hdr_val(TfwStr *hdr, int id, TfwStr *val)
 
 	BUG_ON(TFW_STR_PLAIN(hdr));
 	BUG_ON(TFW_STR_DUP(hdr));
-	BUG_ON(nlen >= hdr->len);
+	BUG_ON(nlen > hdr->len);
```

Comment:
When headers w/o spaces and any values are valid? If a header has length exactly as from hdr_lens, it means that its total length is just a header name plus ":". Moreover, this is used only for special header and it seems this is invalid case.